layouts/joomla/toolbar/basic.php

```diff
@@ -10,11 +10,12 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
 use Joomla\CMS\Language\Text;
 
 Factory::getDocument()->getWebAssetManager()
-	->useScript('core')
-	->useScript('webcomponent.toolbar-button');
+		->useScript('core')
+		->useScript('webcomponent.toolbar-button');
```

Comment:
```suggestion
	->useScript('core')
	->useScript('webcomponent.toolbar-button');
```
Fix PHP CS for Drone.